src/app/App.tsx

```diff
@@ -52,28 +59,63 @@ class App extends Component<AppProps, AppState> {
         });
     }
 
+    /**
+     * Minimize window.
+     */
+    public minimize(): void {
+        remote.getCurrentWindow().minimize();
+    }
+
+    /**
+     * Maximize/unmaximize window.
+     */
+    public maximize(): void {
+        if (!fullScreen) {
```

Comment:
`fullScreen` is still not behaving as it should since a user can enter fullScreen and resize the screen without triggering this function. `fullScreen` should be updated each time the window gets resized or you dont rely on a variable but calculate it here each time.

Reply:
updated!